spec/integration/_dev/deploy/agent/custom-agent.spec.yml

```diff
@@ -13,12 +13,47 @@ spec:
         anyOf:
           - required:
             - hostname
+    volume:
```

Comment:
Do we want to allow all kind of volumes?

Reply:
I am only using a basic named volume in my PR. But I thought it would be easier and faster to just copy the complete schema from docker compose for volumes. Though I'm a little unclear on the motivation behind linting this file in the first place 🤷

Reply:
The reason to validate this file would be to control the features that can be used, for example we don't want to allow to override the hostname, and it could be also problematic to support any network.

I would suggest to remove the properties that we don't plan to use by now.